### PR TITLE
Kill trailing space in template and Update README

### PR DESCRIPTION
**Pull request by JuanitoFatas**
The generator generates comments too. Reflect that in README's template.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ PartyFoul.configure do |config|
   # The OAuth token for the account that is opening the issues on GitHub
   config.oauth_token            = 'abcdefgh1234567890'
 
-  # The API endpoint for GitHub. Unless you are hosting a private
+  # The API api_endpoint for GitHub. Unless you are hosting a private
```
Comment (bcardarella) [inline README.md]: I don't see the reason to change this to `api_endpoint`. That sentence isn't referencing the specific attribute but the nature of it

Comment (JuanitoFatas) [inline README.md]: Then should fix in template: 5ac92bc. :blush:

Comment (bcardarella) [inline README.md]: Ah, I see you were making it consistent. I apologize, I missed that. Yes, thank you for fixing in the template.

**Comment by JuanitoFatas** [comment]
CI error seems unrelated :sweat_smile: 

```
An error occurred while installing
party_foul (1.5.5), and Bundler cannot
continue.
Make sure that `gem install party_foul
-v '1.5.5'` succeeds before bundling.
```

**Comment by JuanitoFatas** [comment]
rebased and passed, yay :tada:

**Comment by bcardarella** [comment]
Thanks!

**Pull request merged**